lua/entities/base_gui_panel/cl_init.lua

```diff
@@ -83,12 +83,15 @@ function ENT:Draw()
 end
 
 function endDrawTimer(ent)
+	if !IsValid( ent ) then return end
```

Comment:
For new code onwards, `not` and other official Lua syntax must be used. `!` is nonstandard GLua syntax and is in the process of being eradicated completely from the project.